docker/config_template.yaml

```diff
@@ -101,18 +101,18 @@ ringpop:
     {{- end }}
     {{- else }}
     bootstrapHosts:
-        - {{ .Env.HOST_IP }}:7933
-        - {{ .Env.HOST_IP }}:7934
-        - {{ .Env.HOST_IP }}:7935
-        - {{ .Env.HOST_IP }}:7939
+        - {{ .Env.HOST_IP }}:{{ default .Env.FRONTEND_PORT "7933" }}
+        - {{ .Env.HOST_IP }}:{{ default .Env.HISTORY_PORT "7934" }}
+        - {{ .Env.HOST_IP }}:{{ default .Env.MATCHING_PORT "7935" }}
+        - {{ .Env.HOST_IP }}:{{ default .Env.WORKER_PORT "7939" }}
     {{- end }}
     maxJoinDuration: 30s
 
 services:
     frontend:
         rpc:
-            port: 7933
-            bindOnIP: {{ default .Env.BIND_ON_IP "127.0.0.1" }}
+            port: {{ default .Env.FRONTEND_PORT "7933" }}
+            bindOnIP: {{ default .Env.FRONTEND_SERVICE "127.0.0.1" }}
```

Comment:
ooops, you misunderstood my comment. I didn't mean to change this. `BIND_ON_IP` is correct here.